tensorboard/plugins/graph/polymer3/tf_graph_app/tf-graph-app.html

```diff
@@ -15,187 +15,6 @@
-          <tf-graph-board
-            id="graphboard"
-            graph-hierarchy="[[graphHierarchy]]"
-            graph="[[graph]]"
-            progress="[[_progress]]"
-            color-by="[[colorBy]]"
-            color-by-params="{{colorByParams}}"
-            render-hierarchy="{{_renderHierarchy}}"
-            selected-node="{{selectedNode}}"
-            trace-inputs="[[_traceInputs]]"
-          ></tf-graph-board>
-        </div>
-      </div>
-    </div>
-  </template>
-</dom-module>
-
-<script>
-  (function() {
-    Polymer({
-      is: 'tf-graph-app',
-      properties: {
-        // To use tf-graph-app, specify one of these 2 properties. Provide either
-        // 1. The path to a pbtxt file to load (pbtxtFileLocation). This option nicely makes the
-        //    progress bar include the time it takes to load the file across the network. The path could
-        //    be either a relative path or an absolute URL (of a resource that supports CORS).
-        // 2. The raw contents of a pbtxt file (pbtxt).
-        // Do not set both of these 2 properties.
-        pbtxtFileLocation: {
-          type: String,
-          observer: '_updateGraph',
-        },
-        pbtxt: {
-          type: String,
-          observer: '_updateGraph',
-        },
-
-        width: {
-          type: Number,
-          observer: '_updateWidth',
-        },
-        height: {
-          type: Number,
-          observer: '_updateHeight',
-        },
-        toolbar: {
-          type: Boolean,
-          observer: '_updateToolbar',
-        },
-        selectedNode: {
-          type: String,
-          notify: true,
-        },
-
-        _renderHierarchy: Object,
-        _progress: Object,
-        _traceInputs: Boolean,
-      },
-      _updateToolbar: function() {
-        this.$$('.container').classList.toggle('no-toolbar', !this.toolbar);
-      },
-      _updateWidth: function() {
-        this.$$('.container').style.width = this.width + 'px';
-      },
-      _updateHeight: function() {
-        this.$$('.container').style.height = this.height + 'px';
-      },
-      _updateGraph: function() {
-        if (this.pbtxtFileLocation) {
-          // Fetch a pbtxt file. The fetching will be part of the loading sequence.
-          this.$.loader.datasets = [
-            {
-              // Just name the dataset based on the file location.
-              name: this.pbtxtFileLocation,
-              path: this.pbtxtFileLocation,
-            },
-          ];
-          this.$.loader.set('selectedDataset', 0);
-        } else if (this.pbtxt) {
-          // Render the provided pbtxt.
-          var blob = new Blob([this.pbtxt]);
-
-          // TODO(@chihuahua): Find out why we call a private method here and do away with the call.
-          this.$.loader._parseAndConstructHierarchicalGraph(null, blob);
-        }
-      },
-      _fit: function() {
-        this.$$('#graphboard').fit();
-      },
-    });
-  })();
-</script>
+<body>
```

Comment:
If I'm understanding correctly, that seems fine with me.  e.g. update the .html to import roboto, keep a `tf_web_library` around in case we want a `tensorboard_html_binary` devserver to use it, and remove the existing `tensorboard_html_binary`.  Please let me know if I misunderstood.